guides/common/modules/proc_performing-discovery-in-pxeless-mode.adoc

```diff
@@ -0,0 +1,56 @@
+[id="performing-discovery-in-pxeless-mode_{context}"]
+= Performing Discovery in PXE-less mode
+[[Implementing_PXE_less_Discovery_{context}]]
+
+Discovery in PXE-less mode uses the Discovery ISO and requires you to attend to the process.
+
+.Prerequisites
+ifdef::satellite,orcharhino[]
+* You have installed the Foreman Discovery image.
+For more information, see xref:Installing_the_Discovery_Service_{context}[].
+endif::[]
+ifndef::satellite,orcharhino[]
+* You have installed the Foreman Discovery image or built a custom Discovery image ISO.
+For more information, see one of the following resources:
+** xref:Installing_the_Discovery_Service_{context}[]
+** xref:Building_a_custom_Discovery_Image_{context}[]
+endif::[]
+
+.Procedure
+. Copy the Discovery ISO to a CD, DVD, or a USB flash drive.
+For example, to copy to a USB drive at `/dev/sdb`:
++
+[options="nowrap" subs="+quotes"]
+----
+# dd bs=4M \
+if=/usr/share/foreman-discovery-image/foreman-discovery-image-_version_.iso \
+of=/dev/sdb
+----
+. Insert the Discovery boot media into a host, start the host, and boot from the media.
+. The Discovery image displays options for either *Manual network setup* or *Discovery with DHCP*:
+
+* *Manual network setup*:
+include::snip_step-discovery-pxeless-primary-interface.adoc[]
+.. On the *Network configuration* screen, enter the *Address*, *Gateway*, and *DNS*.
+Hit *Next* to continue.
+* *Discovery with DHCP*:
+include::snip_step-discovery-pxeless-primary-interface.adoc[]
+.. The Discovery image attempts to automatically configure the network interface by using a DHCP server, such as one that a {SmartProxyServer} provides.
+. On the *Credentials* screen, enter the following options:
++
+** In the *Server URL* field, enter the URL of {ProjectServer} or Discovery {SmartProxyServer}.
+If you refer to a {SmartProxyServer}, include the {SmartProxy} port number.
+** In the *Connection type* field, select the connection type: `Server` for {ProjectServer} or `Foreman Proxy` for {SmartProxyServer}.
+
++
+Hit *Next* to continue.
```

Comment:
```suggestion
Click *Next* to continue.
```

Reply:
This is a problem. The GUI, on which this is done, doesn't have the mouse cursor. The user has to select the button by using the keyboard and then press Enter.